src/connectors/connector_cloudfunctions.workflows.json

```diff
@@ -107,24 +107,24 @@
   "assert_response": {
     "params": [
       "expected_response",
-      "got_response"
+      "actual_response"
     ],
     "steps": [
       {
         "compare": {
           "switch": [
             {
-              "condition": "${expected_response == got_response}",
```

Comment:
no action, but the reason you will see got/want semantics is b/c this is standard in Golang. Fair to make it more generic.

Reply:
Ah, ok, TIL